src/services/mentors.js

```diff
@@ -1499,58 +1457,38 @@ module.exports = class MentorsHelper {
 			}
 
 			const mentorIds = extensionDetails.data.map((item) => item.user_id)
```

Comment:
Multiple passes over extensionDetails.data can be merged into a single loop to extract mentorIds, uniqueOrganizations, and organizationCodes.